examples/paymentbot.py

```diff
@@ -21,41 +21,44 @@
 logging.basicConfig(
     format="%(asctime)s - %(name)s - %(levelname)s - %(message)s", level=logging.INFO
 )
-# set higher logging level for httpx to avoid all GET and POST requests being logged
+# Set a higher logging level for httpx to avoid logging every GET and POST request
```

Comment:
this comment is used in all examples. If it's changed, I would like to have it adapted in the other examples as well

Reply:
I'm unsure whether I should revert it or apply the same change to all the other examples.

Reply:
Your call :) I'm fine with either solution. Thanks for the other updates!

Reply:
Well, I've changed it back to the original because I believe both versions explain the purpose well.